### DISCUSSION TITLE
Is there a command to undo git init?

### DISCUSSION THREAD

**Discussion by MoviesKatta62** [community/community#118542] [Codespaces] [answered]
### Select Topic Area

Bug

### Body

I just Git init'ed a whole desktop and want to undo it. Is there any command for this? Do I actually have to go in and edit the .git directory?

**Comment by LuciferMorningstar33** [comment, marked as answer]
If you are Windows User try going to Desktop Directory using cmd(remember to run as administrator  ==> V.imp) type dir 
If you see .git folder then congratulations you have understood your problem 
Now to tackle it just type

 `del .git`

And if to use Linux/Ubuntu
try You can just delete .git. Typically:

`rm -rf .git`


Then, recreate as the right user.
